get_ip_config_for_iface() python3 error

In `dhcpserver.get_ip_config_for_iface()`:

Iface is a string, and on python3:
```
    ifname = struct.pack('256s', iface[:15])
E   struct.error: argument for 's' must be a bytes object
```

Suggested fix:
```python
ifname = struct.pack('256s', iface[:15].encode())
```

--- a/ptftplib/dhcpserver.py
+++ b/ptftplib/dhcpserver.py
@@ -100,7 +100,7 @@ def get_ip_config_for_iface(iface):
 
     import fcntl
     s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-    ifname = struct.pack('256s', iface[:15])
+    ifname = struct.pack('256s', iface[:15].encode())
     ip = fcntl.ioctl(s.fileno(), SIOCGIFADDR, ifname)
     mask = fcntl.ioctl(s.fileno(), SIOCGIFNETMASK, ifname)
     mac = fcntl.ioctl(s.fileno(), SIOCGIFHWADDR, ifname)

